samples/sampler/stories/qa/VirtualGridList.js

```diff
@@ -129,7 +130,8 @@ class ButtonAndVirtualGridList extends Component {
 	constructor (props) {
 		super(props);
 		this.state = {
-			isPopup: false
+			isPopup: false,
+			rtl: PropTypes.bool
```

Comment:
`rtl` shouldn't be a state as i18nContextDecorator pass as a prop. You can find similar samples in the sampler